src/Worlds.js

```diff
@@ -129,6 +141,83 @@ const worlds: {|
     'Sketchpad': WorldProperties,
     'Space': WorldProperties
 |} = {
+    'AmusementPark': {
+        background: AmusementPark,
+        backgroundGray: AmusementParkGray,
+        backgroundContrast: AmusementParkContrast,
+        backgroundInfo: {
+            A1: 'entrance',
+            C1: 'gameBooth',
+            D1: 'gameBooth',
+            G1: 'pirateShip',
+            H1: 'pirateShip',
+            J1: 'swingRide',
+            K1: 'swingRide',
+            L1: 'swingRide',
+            A2: 'entrance',
+            C2: 'gameBooth',
+            D2: 'gameBooth',
+            G2: 'pirateShip',
+            H2: 'pirateShip',
+            J2: 'swingRide',
+            K2: 'swingRide',
+            L2: 'swingRide',
+            A3: 'merryGoRound',
+            B3: 'merryGoRound',
+            G3: 'pirateShip',
+            H3: 'pirateShip',
+            A4: 'merryGoRound',
+            B4: 'merryGoRound',
+            E4: 'waterSlide',
+            F4: 'waterSlide',
+            J4: 'goKart',
+            K4: 'goKart',
```

Comment:
"go kart" singular would refer to a single vehicle. And on K4 there isn't a kart. We could either distinguish between "go kart" (for the cells with karts on them) and "go kart track" for the cells without, or if we want a single name for all cells then either "go karts" plural or "go kart track".